ehitchhiking-ui/src/app/shared/components/trips-modal/trips-modal.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="trips-modal-container">
```

Comment:
pls resolve conflicts and use common styles for all modals

Reply:
okay